cmd/peer-rest-client.go

```diff
@@ -58,7 +58,7 @@ func (client *peerRESTClient) call(method string, values url.Values, body io.Rea
 // after verifying format.json
 func (client *peerRESTClient) callWithContext(ctx context.Context, method string, values url.Values, body io.Reader, length int64) (respBody io.ReadCloser, err error) {
 	if client == nil || !client.IsOnline() {
-		return nil, errPeerNotReachable
+		return nil, fmt.Errorf("%s - %w", client, errPeerNotReachable)
```

Comment:
This could potentially be returned from calls, so I am a bit hesitant to do this, since it could expose internal data.  I will leave it for @harshavardhana to decide.

Reply:
@jiuker please add `client.Endpoint()` instead of just `client`